includes/Checker/Checks/Abstract_PHP_CodeSniffer_Check.php

```diff
@@ -81,14 +82,24 @@ final public function run( Check_Result $result ) {
 		// Backup the original command line arguments.
 		$orig_cmd_args = $_SERVER['argv'] ?? '';
 
+		$args = $this->get_args( $result );
+
+		// Reset PHP_CodeSniffer config.
+		$this->reset_php_codesniffer_config();
+
+		// Get current installed_paths config.
+		$installed_paths = Config::getConfigData( 'installed_paths' );
```

Comment:
Why does PHPStan complain about `Config` not being found, but not about the `Runner` class? 🤔